plugins/dev9ghzdrk/smap.cpp

```diff
@@ -416,10 +415,11 @@ smap_read16(u32 addr)
 		}
 		*/
 	}
-
+	else
+		return rv;
```

Comment:
This will prevent the logging code below from running even when DEV9_LOG_ENABLE is defined.